### PR TITLE
Changes to refer to the new resourcemover folder from azure-rest-api-specs repo, Added support for BulkRemoveMoveResources Api

### PR DESCRIPTION
**Pull request by ajaygit158**
* [x] Swagger review PR(s) [link1](https://github.com/Azure/azure-rest-api-specs/pull/11131), [link 2](https://github.com/Azure/azure-rest-api-specs/pull/10481), [link3](https://github.com/Azure/azure-rest-api-specs/pull/11052)
* [x]  Metadata file `eng/mgmt/mgmtmetadata_{RP}.txt` is updated
* [x]  Versions in `*.csproj` and `AssemblyInfo.cs` are updated, and follows semver
* [x]  Release notes in `*.csproj` is updated
* [x]  If there's a API-version update, the API tags in `Az.Sdk.props` are updated : No API version updated
* [x]  Generated code is not touched
* [x]  Proper test coverage

### CODE REVIEW

**Comment by YalinLi0312** [comment]
@ajaygit158 Can you rephrase this PR title and follow the checklist below?
- [ ] Swagger review PR(s) link in PR description, and the PR(s) is merged
- [ ] Metadata file `eng/mgmt/mgmtmetadata_{RP}.txt` is updated
- [ ] Versions in `*.csproj` and `AssemblyInfo.cs` are updated, and follows semver
- [ ] Release notes in `*.csproj` is updated
- [ ] If there's a API-version update, the API tags in `Az.Sdk.props` are updated
- [ ] Generated code is not touched
- [ ] Proper test coverage

**Comment by ajaygit158** [comment]
@YalinLi0312 I have added the checklist you recommended. Kindly have a look. The Check Enforcer seems to be stuck. Let me know if any updates are required.

**Comment by YalinLi0312** [comment]
> @YalinLi0312 I have added the checklist you recommended. Kindly have a look. The Check Enforcer seems to be stuck. Let me know if any updates are required.

Can you also update this PR title?

**Comment by YalinLi0312** [comment]
@ajaygit158 To make check enforcer work, you should run scripts(https://github.com/Azure/azure-sdk-for-net/blob/master/eng/scripts/Update-Mgmt-Yml.ps1) to add entries to the mgmt pipeline yml(https://github.com/Azure/azure-sdk-for-net/blob/master/eng/pipelines/mgmt.yml).

**Review by YalinLi0312 — APPROVED** [review]
